lib/rubygems/dependency_installer.rb

```diff
@@ -266,6 +267,7 @@ def find_gems_with_sources(dep, best_only=false) # :nodoc:
 
     set
   end
+  deprecate :find_gems_with_sources, :none, 2019, 12
```

Comment:
this is used in `add_found_dependencies` method

Reply:
Yeah, but `add_found_dependencies` is deprecated too!

Reply:
We should skip deprecations when `add_found_dependencies` calls this method though. I'll add that change.

Reply:
It's also called on line 173 of:
https://github.com/rubygems/rubygems/blob/8d5490364ac3a50eecefe880cbf1201ba355dfb0/lib/rubygems/dependency_installer.rb#L167-L177

But that method (`available_set_for`) seems to only be called by `test_gem_dependency_installer.rb`...

Reply:
Yeah, I didn't count the unit tests for these methods as actual usages of them.

I added the change to not print deprecations to users for methods they are not using directly :point_right: 04719726cba0d2840956d30ff2901f462a69f0a7.